pippo-session-ehcache2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I don't think that we need this file for each module. Whole the project is under APACHE license so I think that it's enough.